go/arrow/internal/arrjson/arrjson.go

```diff
@@ -1090,7 +1091,7 @@ func arrayFromJSON(mem memory.Allocator, dt arrow.DataType, arr Array) arrow.Arr
 	case arrow.BinaryViewDataType:
 		valids := validsToBitmap(validsFromJSON(arr.Valids), mem)
 		nulls := arr.Count - bitutil.CountSetBits(valids.Bytes(), 0, arr.Count)
-		headers := stringHeadersFromJSON(mem, !dt.IsUtf8(), arr.Data)
+		headers := stringHeadersFromJSON(mem, !dt.IsUtf8(), arr.Views)
```

Comment:
was this a change in the integration testing spec?

Reply:
Nope, will add

Reply:
https://github.com/apache/arrow/pull/39019/commits/b9237669d321e38bdeeffb1e675be5399a9575e8